playbooks/roles/jenkins/defaults/main.yml

```diff
@@ -126,7 +126,12 @@ jenkins_list_views:
       - update-credentials
       - rotate-api-tokens
       - rotate-production-notify-callback-token
-      - virus_scan_s3_bucket
+      - virus-scan-s3-buckets-preview
+      - virus-scan-s3-buckets-staging
+      - virus-scan-s3-buckets-production
+      - virus-scan-s3-buckets-nightly-preview
+      - virus-scan-s3-buckets-nightly-staging
+      - virus-scan-s3-buckets-nightly-production
```

Comment:
So we have nightly jobs - when would we use the other ones?

Reply:
I'm using the other ones right now. When we want to run them with custom parameters basically. Big scan jobs etc.